lib/npmrc.js

```diff
@@ -0,0 +1,72 @@
+var asp = require('rsvp').denodeify;
+var fs = require('graceful-fs');
+var path = require('path');
+var utils = require('./utils');
+
+function getFilepath() {
+  var home = process.env.HOME || process.env.USERPROFILE || process.env.HOMEPATH;
```

Comment:
No, it's just the legacy logic that was already here. Will add the extra paths when I add cascading